package.json

```diff
@@ -4,29 +4,29 @@
   "description": "Lunie is the staking and governance platform for proof-of-stake blockchains.",
   "author": "Lunie International Software Systems Inc. <hello@lunie.io>",
   "scripts": {
-    "start": "npm run testnet:start & npm run serve",
     "serve": "kill-port 9080 && vue-cli-service serve --port 9080",
-    "serve:dist": "live-server --https=node_modules/live-server-https --port=9080 --no-browser ./dist",
-    "serve:ios": "npx cap open ios",
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
-    "lint:style": "vue-cli-service lint:style",
     "certificates": "docker-compose -f tasks/certificates/docker-compose.yml up",
     "changelog": "simsala log --pending-path ./changes",
+    "lint:style": "vue-cli-service lint:style",
     "proxy": "node ./tasks/local-testnet/proxy.js",
     "release": "git checkout develop & git pull & git push origin develop:release",
+    "release:android": "git checkout develop && node ./tasks/bump-android-version.js && git commit android/app/build.gradle -m 'updated android app version' && git push origin develop:release-android",
+    "release:ios": "git checkout develop && node ./tasks/bump-ios-version.js && git commit ios/App/App/Info.plist -m 'updated ios app version' && git push origin develop:release-ios",
+    "serve:dist": "live-server --https=node_modules/live-server-https --port=9080 --no-browser ./dist",
+    "serve:ios": "npm run serve && npx cap open ios",
```

Comment:
does this make sense? The simulator is not using the served website as far as I know.

Reply:
it is - because of: https://github.com/luniehq/lunie/pull/3067/files#diff-f4dba1b5ec6bb8ad8372899ef6a9b288R8